From 6d436cbf027259d949bef02ae210ed0e97f61724 Mon Sep 17 00:00:00 2001
From: Chris Burr <chrisburr@users.noreply.github.com>
Date: Mon, 11 Feb 2019 06:13:36 +0000
Subject: [PATCH] Workaround for bad cflags

---
 root_numpy/setup_utils.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/root_numpy/setup_utils.py b/root_numpy/setup_utils.py
index 0bae303..1e3f789 100644
--- a/root_numpy/setup_utils.py
+++ b/root_numpy/setup_utils.py
@@ -83,6 +83,8 @@ def root_flags(root_config='root-config'):
     if sys.version > '3':
         root_cflags = root_cflags.decode('utf-8')
         root_ldflags = root_ldflags.decode('utf-8')
+    # FIXME: Conda's compilers only support -std=c++1z
+    root_cflags = root_cflags.replace('-std=c++17', '-std=c++1z')
     return root_cflags.split(), root_ldflags.split()
 
 
